src/browser/vdiff.js

```diff
@@ -43,6 +45,10 @@ function findLargestRect(elems, margin) {
 			}
 		});
 	});
+	// no elements had a size, so pick a reasonable rect
+	if (!foundElemRect) {
+		largestRect = { left: margin, top: margin, right: margin, bottom: margin };
```

Comment:
`largestRect = { left: 0, top: 0, right: DEFAULT_MARGIN, bottom: DEFAULT_MARGIN };` might make more sense, in case they've set `margin` to 0?  But also, that'll change the `vdiff`, so maybe not worth it lol...